lib/winrm/version.rb

```diff
@@ -0,0 +1,4 @@
+module WinRM
+  # The version of the WinRM library
+  VERSION = '1.3.7'
```

Comment:
Lets go ahead and bump the version to 1.4.0 since we're adding some new functionality, albeit minor.